### PR TITLE
Include a packaging status banner in the README

### PR DESCRIPTION
**Pull request by ottok**
[Packaging status](https://repology.org/project/rdiff-backup/versions)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,3 +234,8 @@ sudo dnf install rdiff-backup
 ```
 sudo pip3 install rdiff-backup --pre
 ```
+
+
+## Packaging status in distros
```
Comment (ottok) [inline README.md:239]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/rdiff-backup/rdiff-backup/pullRequest?prid=5809597)

**Comment by ericzolf** [comment]
Looks neat, is at the end of the readme, doesn't disturb. Fine with me.

**Comment by ericzolf** [comment]
Please add a DOC: comment explaining what you've changed for the release notes. Feel free to squash/merge yourself (you can set the squash commit message yourself in the UI). Thanks.

**Comment by ottok** [comment]
The README is not part of the release, this is just relevant for the Github page. I don't think that it needs to be mentioned in the release notes.

**Comment by ericzolf** [comment]
As you wish...

**Pull request merged**